components/squarespace/sources/new-create-product/new-create-product.mjs

```diff
@@ -0,0 +1,30 @@
+import squarespace from "../../squarespace.app.mjs";
+
+export default {
+  name: "New Create Product",
+  version: "0.0.1",
+  key: "squarespace-new-create-product",
+  description: "Emit new event for each product created.",
+  type: "source",
+  dedupe: "unique",
+  props: {
+    squarespace,
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 5,
+      },
+    },
+  },
+  async run() {
+    const products = await this.squarespace.getProducts();
```

Comment:
I guess we cannot use webhooks here, right? While employing polling, is not `this.squarespace.getProducts` retrieving all products? Please correct me if I am wrong. If it retrieves all products, can we add some logic to filter new products?

Reply:
Maybe I can add a filter, I will take a look.